source/community/release_notes/redwood.rst

```diff
@@ -13,6 +13,47 @@ These are the release notes for the Redwood release, the 18th community release
 Breaking Changes
```

Comment:
Should we include this known issue in this section? https://github.com/openedx/wg-build-test-release/issues/380. 

Basically, this flag worked for the legacy Studio Home, improving performance by adding an organization search bar. But as the report mentions, it's not working correctly in the Studio Home MFE. The good thing is that this flag is not needed in the Studio Home MFE since we already have a search bar and pagination, which improves performance for those home pages with lots of courses. I wouldn't consider it a "breaking change" but a known issue for those who want to turn on the Studio Home MFE and are using the flag. 

Either way, I reported [this issue](https://github.com/openedx/frontend-app-course-authoring/issues/1110) to the Course Authoring maintainers so they're aware.

Reply:
This issue is in this board too: https://github.com/orgs/openedx/projects/28/views/16. I don't know if we should add each issue or link to the board.

Reply:
Again I have no issue with this. I'd favor linking the board unless someone like you can tell me what to include and what not.

I'm just an overpaid secretary, I don't really understand much of what I'm working with here except in the most general (e.g. I know Python and Django) terms.

Reply:
OK.

I'm going to create a Known Issues section with the link to that board you referenced.

I'll also include this in the Breaking section.